frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingSdkOptionCard/EmbeddingSdkOptionCard.tsx

```diff
@@ -39,19 +40,15 @@ export function EmbeddingSdkOptionCard({ onToggle }: EmbeddingOptionCardProps) {
       }
       description={t`Interactive embedding with full, granular control. Embed and style individual Metabase components in your app, and tailor the experience to each person. Allows for CSS styling, custom user flows, event subscriptions, and more. Only available with SSO via JWT.`}
     >
-      <Flex align="center" w="100%">
+      <Group position="apart" align="center" w="100%">
```

Comment:
We don't need to set a margin on the switch if we have the flex container, we can just use `justify-content: space-between` (which Mantine calls `position="apart"` for some reason)

Reply:
what a weird choice from Mantine...

Reply:
it looks like in v7 they actually changed it back to `justify` so we're going to have a great time upgrading things back to what they should have been 🫠